Makefile

```diff
@@ -292,27 +311,47 @@ clean-coverage:
 
 # Building Docker Images for our executables
 ############################################
-images: user-broker-image \
-    controller-manager-image apiserver-image
+images: user-broker-image controller-manager-image apiserver-image
+
+images-all: $(addprefix sub-images-,$(ALL_ARCH))
+sub-images-%:
+	$(MAKE) ARCH=$* images
 
 define build-and-tag # (service, image, mutable_image, prefix)
 	$(eval build_path := "$(4)build/$(1)")
 	$(eval tmp_build_path := "$(build_path)/tmp")
 	mkdir -p $(tmp_build_path)
 	cp $(BINDIR)/$(1) $(tmp_build_path)
-	docker build -t $(2) $(build_path)
+	cp $(build_path)/Dockerfile $(tmp_build_path)
+	# -i.bak is required for cross-platform compat: https://stackoverflow.com/questions/5694228/sed-in-place-flag-that-works-both-on-mac-bsd-and-linux
```

Comment:
minor thing but can you wrap this long line - some of us really like 80 col files :-)